### DISCUSSION TITLE
Could we get better IO performance?

### DISCUSSION THREAD

**Discussion by fungiboletus** [community/community#41484] [Codespaces]
### Select Topic Area

Product Feedback

### Body

I have noticed that Codespaces is still super slow because of its poor IOs performances. I know it's because it's running on Microsoft Azure, that always has been a disaster when it comes to IO performances.

Still, I hope the situation can be improved because it's impossible to recommend Codespaces when a small laptop is magnitude fasters.

**Comment by RyanJamesCaldwell** [comment]
Hi there :wave:

Sorry to hear that you've been experiencing some performance issues!

What specific issues are you running into as it relates to slowness? For example, connecting to a Codespace, typing latency, etc.

In any case, I'd recommend that you verify that you have a strong internet connection. In the meantime, I've created an issue on our internal board but can't guarantee when it'll get prioritized.
